src/vision/cfg/rqt_params_out.yaml

```diff
@@ -0,0 +1,27 @@
+!!python/object/new:dynamic_reconfigure.encoding.Config
```

Comment:
Funky, what is this doing? At the very least we should add a quick comment here.

Reply:
This file is saved by ros's dynamic reconfigure package, if we add a comment ros will just overwrite it when it's saved.